### PR TITLE
 Joesphus problem for recursion added

### PR DESCRIPTION
**Pull request by riyavaryani**

### CODE REVIEW

**Comment by GauravWalia19** [comment]
Please create a issue with hacktoberfest20 template

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
Update the README also.

**Comment by riyavaryani** [comment]
changes done

File: ARRAYS/C++/Recursion/josephusProblem.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    if (n == 1) 
+        return 1; 
+    else
+       
+        return (josephus(n - 1, k) + k-1) % n + 1; 
+} 
+  
+int main() 
+{ 
+    int n;
```
Comment (GauravWalia19) [inline ARRAYS/C++/Recursion/josephusProblem.cpp:22]: Please use good variable names like 
**int n  -> int noOfPeople**

File: ARRAYS/C++/Recursion/josephusProblem.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cin>>n; 
+    int k;
+    cin>>k;
```
Comment (GauravWalia19) [inline ARRAYS/C++/Recursion/josephusProblem.cpp:25]: int k -> int kthPeopleToKill

File: ARRAYS/C++/Recursion/josephusProblem.cpp [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+The problem is explained here-
+There are n people standing in a circle waiting to be executed. The counting out begins at some point in the circle and proceeds around the circle in a fixed direction. In each step, a certain number of people are skipped and the next person is executed. The elimination proceeds around the circle (which is becoming smaller and smaller as the executed people are removed), until only the last person remains, who is given freedom. Given the total number of persons n and a number k which indicates that k-1 persons are skipped and kth person is killed in circle. The task is to choose the place in the initial circle so that you are the last one remaining and so survive.
+*/
+
+
+#include <bits/stdc++.h> 
+using namespace std; 
+  
+int josephus(int n, int k) 
+{
```
Comment (GauravWalia19) [inline ARRAYS/C++/Recursion/josephusProblem.cpp:12]: update the variables for this method also

**Comment by GauravWalia19** [comment]
Thanks for choosing our repository for contributing to Hacktober Fest 20
As a part of the new process this Issue is marked as hacktoberfest-expiry & closed
Please check our new updated contribution guidelines [here](https://github.com/TechOUs/DSKaKhel/blob/master/CONTRIBUTING.md)

**Pull request closed without merging**